src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -124,6 +124,11 @@ public String getCustomMessage() {
         return customMessage;
     }
 
+	public SlackNotifier()
+	{
+		
+	}
+	
     @DataBoundConstructor
     public SlackNotifier(final String teamDomain, final String authToken, final String room, final String buildServerUrl,
```

Comment:
Why empty constructor when this `SlackNotifier(final String teamDomain, ...` constructor can be used in a groovy script?